Classes/Middleware/LanguageRedirectionMiddleware.php

```diff
@@ -36,6 +36,11 @@ public function process(ServerRequestInterface $request, RequestHandlerInterface
             return $handler->handle($request);
         }
 
+        // Do nothing, if accept language header is not set
+        if (! isset($request->getServerParams()['HTTP_ACCEPT_LANGUAGE'])) {
```

Comment:
To avoid a negation and improve readability I suggest to go for `empty()`. What do you think?

```suggestion
        if (empty($request->getServerParams()['HTTP_ACCEPT_LANGUAGE'])) {
```